api/init.go

```diff
@@ -46,12 +50,17 @@ func InitWithConfigFile(configPath string) error {
 
 // initCoreComponents init core components with default config
 // it's better SetDefaultConfig before initCoreComponents
-func initCoreComponents() error {
-	if err := metric.InitTask(); err != nil {
-		return err
+func initCoreComponents(confEntity *config.Entity) error {
+	if confEntity.EnableMetricLog() {
```

Comment:
> Add field EnableMetricLog in config.Entity to indicate whether run metric log task, default false;

This can be achieved when `MetricLogFlushIntervalSec=0`.

> Add field EnableSystemStat in config.Entity to indicate whether run system statistic task, default false;

This can be achieved when `SystemStatCollectIntervalMs=0`.

Reply:
Make sense. the default value of `MetricLogFlushIntervalSec` and `SystemStatCollectIntervalMs` is zero, that meaning not run metric log task and system statistic task.
Right?

Reply:
Updated

Reply:
> Make sense. the default value of `MetricLogFlushIntervalSec` and `SystemStatCollectIntervalMs` is zero, that meaning not run metric log task and system statistic task.
> Right?

DefaultMetricLogFlushIntervalSec   uint32 = 1
DefaultSystemStatCollectIntervalMs uint32 = 1000

Reply:
Sorry for misunderstanding.  It should be "the setting value of MetricLogFlushIntervalSec and SystemStatCollectIntervalMs is zero, that meaning not run metric log task and system statistic task"..